lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -2627,13 +2627,13 @@ printRequirementStub(ValueDecl *Requirement, DeclContext *Adopter,
     Options.FunctionDefinitions = true;
     Options.PrintAccessorBodiesInProtocols = true;
 
+    bool AdopterIsClass = isa<ClassDecl>(Adopter->getAsDecl());
```

Comment:
This isn't going to be the right test; the adopter might be an extension of a class. What was wrong with the previous one?

Reply:
My bad, thanks. To be honest, the previous one seemed inelegant to me.